src/classes/ADDR_Addresses_TEST2.cls

```diff
@@ -466,7 +466,9 @@ public with sharing class ADDR_Addresses_TEST2 {
 
         // run the scheduled batch directly
         Test.startTest();
-        Database.executeBatch(new ADDR_Seasonal_SCHED(), 10);
+        final Integer BATCH_SIZE = (UTIL_CustomSettingsFacade.getHouseholdsSettings().Seasonal_Addresses_Batch_Size__c != null) ?
```

Comment:
- Variables with all capital letters are reserved for constants. Local vars, even if they are `final`, use lowerCamelCase. 
- Let's set the Custom Settings field batch size in the test and then retrieve it in the test. (This comment is applicable to other unit test in this class executing `ADDR_Seasonal_SCHED` batch.)